docs/whatsnew/1.2.rst

```diff
@@ -43,7 +43,11 @@ Lomb-scargle periodograms
 
 Other new statistics features
 -----------------------------
-
+* The Jackknife resampling method is available in `astropy.stats.jackknife`.
+* Circular statistics (circular mean, variance, etc) are provided in
+  `astropy.stats.circstats`.
```

Comment:
Add:  

The circular stats are compatible with `astropy.unit.Quantity` and follow the same conventions as used in R.